packages/ozone/src/mod-service/index.ts

```diff
@@ -322,6 +327,33 @@ export class ModerationService {
       subject.blobCids,
     )
 
+    if (
+      isModEventDivert(event) &&
+      subjectInfo.subjectUri &&
+      subjectInfo.subjectBlobCids?.length
+    ) {
+      const blobDiverts = await Promise.all(
```

Comment:
For other event types we don't take the effect of the event in `logEvent()`.  Instead there's a method that handles it , which gets called within `com.atproto.admin.emitModerationEvent` after logging the event.  For consistency maybe we should move this into that pattern.